doc/api/modules.md

```diff
@@ -984,6 +984,36 @@ const requireUtil = createRequireFromPath('../src/utils/');
 requireUtil('./some-tool');
 ```
 
+### module.syncBuiltinESMExports()
+<!-- YAML
+REPLACEME
```

Comment:
This looks like the problem. You want this instead, I think:

```suggestion
added: REPLACEME
```

Reply:
Ah, I had this PR based with the other PR so assumed the shared bug was on master instead of here. Thanks for pointing this out!